packages/framework/build.xml

```diff
@@ -50,10 +50,10 @@
     </if>
 
     <target name="build" depends="build-deploy-part-1-db-independent,build-deploy-part-2-db-dependent" description="Builds application for production preserving your DB."/>
-    <target name="build-deploy-part-1-db-independent" depends="generate-build-version,clean,redis-check,composer,npm,dirs-create,assets" description="First part of application build for production preserving your DB (can be run without maintenance page)."/>
+    <target name="build-deploy-part-1-db-independent" depends="generate-build-version,clean,composer,redis-check,npm,dirs-create,assets" description="First part of application build for production preserving your DB (can be run without maintenance page)."/>
```

Comment:
hmmm. now i do not know why redis-check is in target named db-independent. Maybe we could remove redis-check from this target

Reply:
I generally want to check Redis availability as soon as possible...

Reply:
but redis is kind of db.

Reply:
Yes, but this target was created not because it should be independent of database, but because i can run during deployment without maintenance page (assuming you're running it in separate directory / container).